app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")

```

Comment:
BTW, It is not so good to create a connection here, I recommend moving it to an argument in init with the default value. Check about Dependency Injection

!Optional to change